qutebrowser/browser/downloads.py

```diff
@@ -580,6 +583,9 @@ def on_reply_error(self, code):
         if code == QNetworkReply.OperationCanceledError:
             return
         else:
+            if self.reply is None:
+                log.downloads.debug("QNetworkReply disappeared %s", self)
+                return
```

Comment:
FWIW, I can also reproduce it on master with `:download http://pagead2.googlesyndication.com/pagead/js/google_top_exp.js`